### PR TITLE
feat(notifier): shows events in view own purchases

### PR DESCRIPTION
**Pull request by jurajpiar**
Finishes the https://rsklabs.atlassian.net/browse/RMKT-633 by resolving events from the subscription data.

https://user-images.githubusercontent.com/36888576/119971912-7a781380-bfa9-11eb-8314-792de6834fea.mov

### CODE REVIEW

File: src/components/pages/notifier/mypurchase/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  const type = (
+    <MarketplaceCell>
+      {eventType}
+    </MarketplaceCell>
+  )
+
+  return ({
+    channels,
+    id: `${name}.${contract}`,
```
Comment (itofarina) [inline src/components/pages/notifier/mypurchase/utils.tsx]: Is `contract` a JSX type? If that's the case, can we use another id?

Comment (jurajpiar) [reply src/components/pages/notifier/mypurchase/utils.tsx]: ah yea. good point. thanks

Comment (jurajpiar) [inline src/components/pages/notifier/mypurchase/utils.tsx]:
```suggestion
    id: `${name}@${contractAddress}`,
```

**Comment by jurajpiar** [comment]
Do squash merge

**Review by itofarina — APPROVED** [review]
Looks great!

**Pull request merged**